### DISCUSSION TITLE
Publishing images to Docker Hub

### DISCUSSION THREAD

**Discussion by beautiful-orca** [martin-ueding/geo-activity-playground#171] [Ideas]
Publishing release versions of docker images to [Docker Hub ](https://hub.docker.com/) makes it easier to spin up this project as docker container via docker compose.
I think a lot of people just want to copy and adjust a `compose.yml` file instead of manually cloning the repo, do a `docker build` and do a `docker run`.

Discoverability of this project might also increase because the self-hosting scene seems to like docker hub to discover new projects and find docker specific instructions.

[Publishing Docker Images](https://docs.github.com/en/actions/use-cases-and-examples/publishing-packages/publishing-docker-images)

**Comment by beautiful-orca** [comment]
Docker images now are posted on Github Container Regristry.
This should be documented and made available as an official install method with docker compose and this image.

Afterwards it could be promoted in for example r/selfhosted on reddit.

**Reply by martin-ueding** [reply]
You seem to use that setup; could you write the documentation for that?

**Reply by beautiful-orca** [reply]
I'm not sure how the docker install fits with the install flow on [Documentation Home](https://martin-ueding.github.io/geo-activity-playground/)

Maybe the docker install needs it own flow by linking to the next steps for getting started.


"Get started"
"Install the software" - can link to the docker install page

"Create a base directory." - that should be a part of the docker compose install steps, as you have to define the path in the docker compose.yaml file already.

"Start the webserver." - is also specific to the docker install

"Choose a method to record activities." - Docker install page can link to [Record Activities](https://martin-ueding.github.io/geo-activity-playground/record-activities/)

And for "Import your activities" link back to home page [Importing your activities](https://martin-ueding.github.io/geo-activity-playground/#importing-your-activities)

**Reply by beautiful-orca** [reply]
Github readme also needs a update to the [Installation methods](https://github.com/martin-ueding/geo-activity-playground?tab=readme-ov-file#%EF%B8%8F-installation).

Add docker in the list.

Also each mentioned platform may directly link to specific install instructions.
